### PR TITLE
Move sidebar menu in a layout

### PR DESCRIPTION
**Pull request by SonaliChavan95**

### CODE REVIEW

File: routes/index.js [hunk 1]
```diff
@@ -14,32 +14,35 @@ module.exports = (app) => {
...
     res.status(200).render('users/admin', { title: 'Admin Page' });
   });
   app.get('/receipt', async (req, res) => {
-    res.status(200).render('users/receipt', { title: 'Receipt Page' });
+    res.status(200).render('users/receipt', { title: 'Receipient Page' });
   });
 
   //dynamic paths
   app.get('/donnor', async (req, res) => {
-    res.status(200).render('users/donnor', { title: 'Donnors Page' });
```
Comment (abhinav098) [inline routes/index.js:36]: Sonali you can remove these routes. These have been moved to `users.js` line no 26 to 46.

File: views/layouts/main.handlebars [hunk 2]
```diff
@@ -47,25 +47,8 @@
...
-  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/js/bootstrap.bundle.min.js"
-    integrity="sha384-ho+j7jyWK8fNQe+A12Hb8AhRq26LrZ/JpcUGGOn+Y7RsweNrtN/tE3MoK7ZeZDyx"
-    crossorigin="anonymous"></script>
-  <script src="https://cdn.jsdelivr.net/npm/popper.js@1.16.1/dist/umd/popper.min.js"
-    integrity="sha384-9/reFTGAW83EW2RDu2S0VKaIzap3H66lZH81PoYlFhbGU+6BZp6G7niu735Sk7lN"
-    crossorigin="anonymous"></script>
-  <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.5.3/dist/js/bootstrap.min.js"
-    integrity="sha384-w1Q4orYjBQndcko6MimVbzY0tgp4pWB4lZ7lr30WKz0vr/aWKhXdBNmNb5D92v7s"
-    crossorigin="anonymous"></script>
-  <script src='../../public/js/main.js'></script>
```
Comment (abhinav098) [inline views/layouts/main.handlebars:60]: 👍

File: views/layouts/main2.handlebars [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+      {{>flash_messages message=message sessionMessage=sessionMessage}}
+      <main>
+        <div class="error">
+          {{error}}
+        </div>
+        {{{body}}}
+      </main>
+    </div>
+  </div>
+  {{>layout/script_tags}}
```
Comment (abhinav098) [inline views/layouts/main2.handlebars:78]: Did you forget to add a footer here?

**Review by abhinav098 — CHANGES_REQUESTED** [review]
Looks good, added minor comments.

File: views/layouts/main2.handlebars (hunk 3 above)
Comment (abhinav098) [reply views/layouts/main2.handlebars:78]: Never mind. Ignore this comment. I saw it's in the sidebar.

File: views/layouts/main2.handlebars [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+      <div class="sidebar-header">
+        <h3>Helping Hands</h3>
+      </div>
+      <ul class="list-unstyled components">
+        <li><a href="#"><i class="fa fa-address-card mr-2" aria-hidden="true"></i> Profile</a></li>
+        <li><a href="#"><i class="fa fa-plus-square mr-2" aria-hidden="true"></i> My Donations</a></li>
+        <li><a href="#"><i class="fa fa-boxes mr-2" aria-hidden="true"></i> View Donations</a></li>
+        <li><a href="#"><i class="fa fa-cog mr-2" aria-hidden="true"></i> Settings</a></li>
+        <li><a href="/about"><i class="fa fa-briefcase mr-2" aria-hidden="true"></i> About</a></li>
+        <li><a href="/logout"><i class="fa fa-arrow-alt-circle-left mr-2" aria-hidden="true"></i> Logout</a></li>
```
Comment (abhinav098) [inline views/layouts/main2.handlebars:35]: I think it's `/auth/logout`.

**Review by iamikenna — APPROVED** [review]

**Review by abhinav098 — APPROVED** [review]

**Pull request merged**